heap.py

```diff
@@ -0,0 +1,68 @@
+from dynamic_array import DynamicArray
+class MinHeap:
+    def __init__(self):
+        self.array = DynamicArray(10)
+
+    def add(self, value):
+        self.array.append(value)
+        self.heapify()
+
+    def delete(self):
+        if self.array.is_empty():
+            return False
```

Comment:
Interesting choice! More appropriate way to do would be to throw error as you're trying to delete from EMPTY. Which approach sounds more appropriate to you  @ChinthanBD ?

Reply:
Don't want to throw error. Exiting gracefully returning false seems good